pages/index.js

```diff
@@ -13,15 +13,16 @@ export default function Home({ cards }) {
       ) : null}
       <WrapperDiv>
         <CardList>
-          {cards?.map((card, index) => (
+          {cards?.map((card, id) => (
             <Card
-              key={index}
-              id={index}
-              wish={card.wish}
+              key={id}
+              id={card.id}
```

Comment:
That's the exact same thing as before. You just renamed the `index` paramter to `id`. You need to use `card.id` as key prop.